Engineering-Reference/GoogleDestinationAO.md

```diff
@@ -389,55 +373,55 @@ Default Value: File
 
 Select how file permissions will be applied when migrating to Shared Drive.
 
-Change Conditions: ??
+Change Conditions: None - No file permissions migrated. File - Original file permissions migrated. Root - Any items on file ACLs will be added as members of the Shared Drive.
```

Comment:
Huh?  how can you have no file permissions migrated but original file permissions migrated?

Reply:
None, File and Root are the three options. Will make it clearer.